database_admin/migrations/047_system_package_reverse_idx.down.sql

```diff
@@ -0,0 +1 @@
+DROP INDEX IF EXISTS system_package_package_system_idx;
```

Comment:
This should contain also recreating of previous index and dropping of name_id column.

Reply:
We're not dropping previous indexes